x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_exceptions/utils/helpers.tsx

```diff
@@ -292,55 +292,153 @@ export const lowercaseHashValues = (
 };
 
 /**
- * Returns the value for `file.Ext.code_signature` which
- * can be an object or array of objects
+ * Returns an array of exception entries for either
+ * `file.Ext.code_signature` or 'file.code_signature`
+ * as long as the `trusted` field is `true`.
  */
 export const getFileCodeSignature = (
   alertData: Flattened<Ecs>
-): Array<{ subjectName: string; trusted: string }> => {
+): Array<{ field: string; type: 'nested'; entries: EntriesArray }> => {
```

Comment:
Is this return type correct? The explicit object being returned below doesn't seem to conform to this type.